src/wizards/deployToOpenShift/deployToOpenShift.ts

```diff
@@ -0,0 +1,51 @@
+import { window } from "vscode";
+import { ProjectLabelInfo } from "../../definitions/ProjectLabelInfo";
+import { getOpenShiftConnector, installOpenShiftConnector, isOpenShiftConnectorInstalled } from "../../utils/openShiftConnectorUtils";
+import { getQuarkusProject } from "../getQuarkusProject";
+
+/**
+ * Attempts to select a Quarkus project, then deploy it to OpenShift using OpenShift Connector
+ */
+export async function deployToOpenShift() {
+  try {
+    const openShiftConnectorExtension: any = await installOpenShiftConnectorIfNeeded();
+    const quarkusProject: ProjectLabelInfo = await getQuarkusProject();
+    // openShiftConnectorExtension.Component.createFromFolder(quarkusProject.uri);
```

Comment:
Considering we don't have API for that yet, I would suggest to use vscode 'commands' API. Here is the issue for extension to provide vscode command for that https://github.com/redhat-developer/vscode-openshift-tools/issues/1832.